### PR TITLE
Import virtualbox certificate before installation (fix #1013)

### LINKED ISSUE #1013: Investigate accepting certificate for virtualbox drivers without additional user input
If the system does not have the proper oracle certificate in its trusted authorities, virtualbox installation will present a modal dialog asking the user to accept it. Find a way to avoid the dialog and do it automatically.

### PR DESCRIPTION
**Pull request by jrichter1**
Extract the certificate from virtualbox executable and import it before installing. In case somebody does not have it yet, it should no longer prompt for "trusting oracle" when installing vbox drivers.

### CODE REVIEW

File: browser/model/virtualbox.js [hunk 1]
```diff
@@ -118,9 +118,12 @@ class VirtualBoxInstallWindows extends VirtualBoxInstall {
 
   installAfterRequirements(progress, success, failure) {
     let installer = new Installer(this.keyName, progress, success, failure);
-    return installer.exec(
-      [`"${this.downloadedFile}"`, '--extract', '-path', `"${this.installerDataSvc.virtualBoxDir()}"`, '--silent'].join(' ')
-    ).then(() => {
+    return this.importCertificate(installer
+    ).catch(() => {
+      // continue even if certificate fails to import automatically
```
Comment (dgolovin) [inline browser/model/virtualbox.js]: I don't like empty catch blocks, lets at least log warning here.

File: browser/model/virtualbox.js [hunk 2]
```diff
@@ -136,6 +139,19 @@ class VirtualBoxInstallWindows extends VirtualBoxInstall {
     });
   }
 
+  importCertificate(installer) {
+    let command = [
```
Comment (dgolovin) [inline browser/model/virtualbox.js]: There should be factory method for command.

File: test/unit/model/virtualbox-test.js [hunk 3]
```diff
@@ -162,6 +162,27 @@ describe('Virtualbox installer', function() {
         installer.ipcRenderer = {on: function() {}};
       });
 
+      it('should import the certificate from the executable', function() {
+        sandbox.stub(child_process, 'execFile').yields('done');
+
+        let command = [
```
Comment (dgolovin) [inline test/unit/model/virtualbox-test.js]: Use command factory method here to avoid copy-paste.

**Comment by jrichter1** [comment]
Issues have been addressed, hopefully ;)

**Pull request merged**